packages/events/src/service/config/config.ts

```diff
@@ -10,23 +10,16 @@
  */
 
 import { env } from '@events/environment'
-import { EventConfig, getOrThrow } from '@opencrvs/commons'
-import fetch from 'node-fetch'
-import { array } from 'zod'
+import { createAPIClient, getOrThrow } from '@opencrvs/commons'
+
+const apiClient = createAPIClient(env.COUNTRY_CONFIG_URL)
 
 export async function getEventConfigurations(token: string) {
-  const res = await fetch(new URL('/events', env.COUNTRY_CONFIG_URL), {
+  return apiClient.getEventConfigurations({
```

Comment:
This here is pretty cool – country config now also provides a strictly typed API client to use country config through